test_pr_quick_start_guide.rst

```diff
@@ -0,0 +1,70 @@
+*****************************
+Quick-start development guide
+*****************************
+
+This guide describes all steps needed to test another author's pull request (hereinafter ``PR``).
+
+It can be useful, particularly, for reviewers and issue authors when they want to check if a PR fixes the reported bug.
+
+.. contents:: Topics
+
+Prepare your environment
+========================
+
+We assume that you use Linux as a work environment (you can use a virtual machine as well) and have ``git`` installed.
+
+1. If possible, make sure that you have installed and started ``docker``. While you can also run tests without docker, this makes it a lot easier since you do not have to install the precise requirements, and tests are running properly isolated and in the exact same environments as in CI. You often can also use ``podman`` with the ``docker`` executable shim, so if you have that you probably do not need to install ``docker``.
```

Comment:
```suggestion
1. If possible, make sure that you have installed and started ``docker``. While you can also run tests without docker, this makes it a lot easier since you do not have to install the precise requirements, and tests are running properly isolated and in the exact same environments as in CI. You often can also use ``podman`` with the ``docker`` executable shim, so if you have that you probably do not need to install ``docker``. The latest ansible-core development version also supports the ``podman`` CLI program.
```

Reply:
@aminvakil feel free to adjust the create pr quick start guide similarly

Reply:
Added to `create_pr_quick_start` in https://github.com/ansible/community-docs/pull/16/commits/40200566a4c573e71886dda715759e853b989abc.